gilda/app/__init__.py

```diff
@@ -1,13 +1,36 @@
+"""
+Run the grounding app.
+
+Run as module:
+    `python -m gilda.app --host <host> --port <port> --terms <terms>`
+
+Run with gunicorn:
+    `gunicorn -w <worker count> -b <host>:<port> -t <timeout> gilda.app:gunicorn_app`
+
+For this use case, set the `GILDA_TERMS` environment variable to the path to the
+terms file.
+"""
+
 import argparse
 from .app import get_app
 
 
-def main():
+def parse_args():
     parser = argparse.ArgumentParser(
         description='Run the grounding app.')
     parser.add_argument('--host', default='0.0.0.0')
     parser.add_argument('--port', default=8001, type=int)
     parser.add_argument('--terms')
     args = parser.parse_args()
-    app = get_app(terms=args.terms)
-    app.run(host=args.host, port=args.port, threaded=False)
+    return args
+
+
+if __name__ == '__main__':
+    _args = parse_args()
+    app = get_app(_args.terms)
+    app.run(_args.host, _args.port, threaded=False)
+else:
+    # Assume the terms file is set in the environment
+    import os
+    terms = os.environ.get('GILDA_TERMS')
```

Comment:
Does this default to the standard location if the variable is unset?

Reply:
Yes, the condition to load the default terms in the grounder is `if terms is None`. I tried to set terms path that didn't exist and that triggered an error. `unset GILDA_TERMS` restored the default terms location to be used.